### DISCUSSION TITLE
How can I avoid these spaces?

### DISCUSSION THREAD

**Discussion by gerroon** [kimai/kimai#5061] [Questions & Answers] [answered]
Hi

Is there a way to avoid these double lines? It is gets confusing and uses too much space. thxn
![image](https://github.com/user-attachments/assets/50a8b9b2-58d3-446a-b6a2-b996b4d30a65)

**Comment by kevinpapst** [comment, marked as answer]
This is off by default. You turned "daily statistics" on in your user preferences.
